README.md

```diff
@@ -1,14 +1,2112 @@
+  	<label for="toggle-all">Mark all as complete</label>
+  	<ul class="todo-list">
+      <li data-id="1590167947253" class="">
+        <div class="view">
+          <input class="toggle" type="checkbox">
+          <label>Learn how to build a Todo list in Phoenix</label>
+          <button class="destroy"></button>
+        </div>
+      </li>
+      <li data-id="1590167956628" class="completed">
+        <div class="view">
+          <input class="toggle" type="checkbox">
+          <label>Completed item</label>
+          <button class="destroy"></button>
+        </div>
+      </li>
+    </ul>
+  </section>
+  <footer class="footer" style="display: block;">
+  	<span class="todo-count"><strong>1</strong> item left</span>
+  	<ul class="filters">
+  		<li>
+  			<a href="#/" class="selected">All</a>
+  		</li>
+  		<li>
+  			<a href="#/active">Active</a>
+  		</li>
+  		<li>
+  			<a href="#/completed">Completed</a>
+  		</li>
+  	</ul>
+  	<button class="clear-completed" style="display: block;">Clear completed</button>
+  </footer>
+</section>
+```
+
+Let's convert this `HTML` to an Embedded Elixir
+([`EEx`](https://hexdocs.pm/eex/EEx.html)) template.
+
+
+> **Note**: the _reason_ that we are copying this `HTML`
+from the browser's Elements inspector
+instead of _directly_ from the source
+on GitHub:
+[`examples/vanillajs/index.html`](https://github.com/tastejs/todomvc/blob/c50cc922495fd76cb44844e3b1cd77e35a5d6be1/examples/vanillajs/index.html#L18)
+is that this is a "single page app",
+so the `<ul class="todo-list"></ul>`
+only gets populated in the browser.
+Copying it from the browser Dev Tools
+is the easiest way to get the _complete_ `HTML`.
+
+<br />
+
+### 3.1 Paste the HTML into `index.html.eex`
+
+Open the `lib/app_web/templates/item/index.html.eex` file
+and scroll to the bottom.
+
+Then (_without removing the code that is already there_)
+paste the `HTML` code we sourced from TodoMVC.
+
+> e.g:
+[`/lib/app_web/templates/item/index.html.eex#L33-L73`](https://github.com/dwyl/phoenix-todo-list-tutorial/blob/bddacda93ecd892fe0907210bab335e6b6e5e489/lib/app_web/templates/item/index.html.eex#L33-L73)
+
+If you attempt to run the app now
+and visit
+[http://localhost:4000/items/](http://localhost:4000/items/) <br />
+You will see this (_without the TodoMVC `CSS`_):
+
+![before-adding-css](https://user-images.githubusercontent.com/194400/82725401-af85c200-9cd4-11ea-8717-714477fc3157.png)
+
+That's obviously not what we want,
+so let's get the TodoMVC `CSS`
+and save it in our project!
+
+<br />
+
+### 3.2 Save the TodoMVC CSS to `/assets/css`
+
+Visit
+http://todomvc.com/examples/vanillajs/node_modules/todomvc-app-css/index.css <br />
+and save the file to `/assets/css/todomvc-app.css`.
+
+e.g:
+[`/assets/css/todomvc-app.css`](https://github.com/dwyl/phoenix-todo-list-tutorial/blob/65bec23b92307527a414f77b667b29ea10619e5a/assets/css/todomvc-app.css)
+
+<br />
+
+### 3.3 Import the `todomvc-app.css` in
```

Comment:
I'm not sure about this suggestion but do we want to add where to import the css file?

```suggestion
### 3.3 Import the `todomvc-app.css` in `app.scss`
```